packages/backend/src/modules/dashboard/dashboard.model.ts

```diff
@@ -0,0 +1,42 @@
+import mongoose from 'mongoose'
```

Comment:
why this file is looking like new?

Reply:
Before there was `Dashboard.ts`, now there is `dashboard.model.ts`